lib/_http_outgoing.js

```diff
@@ -464,10 +468,6 @@ OutgoingMessage.prototype.write = function(chunk, encoding, callback) {
       else
         len = chunk.length;
 
-      if (this.connection && !this.connection.corked) {
-        this.connection.cork();
-        process.nextTick(connectionCorkNT, this.connection);
-      }
```

Comment:
by moving this block higher up, `cork()` will happen also for chunked requests. Is this ok? @nodejs/http